frontends/web/src/components/layout/header.tsx

```diff
@@ -15,37 +15,40 @@
  * limitations under the License.
  */
 
-import React, { ReactNode } from 'react';
+import React, { ReactNode, useContext } from 'react';
 import { useTranslation } from 'react-i18next';
 import { share } from '../../decorators/share';
-import { translate, TranslateProps } from '../../decorators/translate';
-import { TSharedProps as SharedPanelProps, store as panelStore, toggle as toggleGuide } from '../guide/guide';
+import { SharedPanelProps, panelStore } from '../sidebar/sidebar';
 import { GuideActive, MenuLight, MenuDark } from '../icon';
 import { toggleSidebar } from '../sidebar/sidebar';
+import AppContext from '../../contexts/AppContext';
 import style from './header.module.css';
-
 interface HeaderProps {
     title?: string | JSX.Element | JSX.Element[];
     narrow?: boolean;
     hideSidebarToggler?: boolean;
     children?: ReactNode;
+    guideExists?: boolean;
```

Comment:
The only place where we use Header prop `guideExists={false}` is in mobile-settings.tsx component.

Is this still needed?

Could the Header component just ask the context if there is a guide (there should be none on mobile-settings)?